scripts/disabled_tests/tests/data/playlist_input.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<playlist>
+	<test>
+		<NewTestCaseName>ParallelStreamsLoadTest_special_J9</NewTestCaseName>
+	</test>
+	<test>
+		<a>
+			<b>
+				<c>
+					<testCaseName>ParallelStreamsLoadTest_special_J9</testCaseName>
+				</c>
+			</b>
+		</a>
```

Comment:
Could you please explain more on why do we need ```<a> <b> <c>``` here?

Reply:
They are simply there to show by example how the parser searches all (grand*)children of a `test` node to find a `testCaseName`.

Added a explanatory comment in acc49cb338dcd7dd67cc06feb4dcce9752cc6f7b